Packet++/header/TcpReassembly.h

```diff
@@ -25,43 +25,61 @@
+ *   the connection (5-tuple, 4-byte hash key describing the connection, etc.) and also a pointer to a "user cookie",
+ *   meaning a pointer to a structure provided by the user during the creation of the pcpp#TcpReassembly instance. The
+ *   end connection callback also provides the reason for closing it ("naturally" or manually).
  *
  * __Basic Usage and APIs:__
  * - pcpp#TcpReassembly c'tor - Create an instance, provide the callbacks and the user cookie to the instance
  * - pcpp#TcpReassembly#reassemblePacket() - Feed pcpp#TcpReassembly instance with packets
  * - pcpp#TcpReassembly#closeConnection() - Manually close a connection by a flow key
  * - pcpp#TcpReassembly#closeAllConnections() - Manually close all currently opened connections
- * - pcpp#TcpReassembly#OnTcpMessageReady callback - Invoked when new data arrives on a certain connection. Contains the new data as well as connection data (5-tuple, flow key)
+ * - pcpp#TcpReassembly#OnTcpMessageReady callback - Invoked when new data arrives on a certain connection. Contains the
+ *   new data as well as connection data (5-tuple, flow key).
  * - pcpp#TcpReassembly#OnTcpConnectionStart callback - Invoked when a new connection is identified
- * - pcpp#TcpReassembly#OnTcpConnectionEnd callback - Invoked when a connection ends (either by FIN/RST or manually by the user)
+ * - pcpp#TcpReassembly#OnTcpConnectionEnd callback - Invoked when a connection ends (either by FIN/RST or manually by
+ *   the user).
  *
  * __Additional information:__
- * When the connection is closed the information is not being deleted from memory immediately. There is a delay between these moments. Existence of this delay is caused by two reasons:
+ * When the connection is closed the information is not being deleted from memory immediately. There is a delay between
+ * these moments. Existence of this delay is caused by two reasons:
  * - pcpp#TcpReassembly#reassemblePacket() should detect the packets that arrive after the FIN packet has been received
- * - the user can use the information about connections managed by pcpp#TcpReassembly instance. Following methods are used for this purpose: pcpp#TcpReassembly#getConnectionInformation and pcpp#TcpReassembly#isConnectionOpen.
- * Cleaning of memory can be performed automatically (the default behavior) by pcpp#TcpReassembly#reassemblePacket() or manually by calling pcpp#TcpReassembly#purgeClosedConnections in the user code.
- * Automatic cleaning is performed once per second.
+ * - the user can use the information about connections managed by pcpp#TcpReassembly instance. Following methods are
+ * used for this purpose: pcpp#TcpReassembly#getConnectionInformation and pcpp#TcpReassembly#isConnectionOpen. Cleaning
+ * of memory can be performed automatically (the default behavior) by pcpp#TcpReassembly#reassemblePacket() or manually
+ * by calling pcpp#TcpReassembly#purgeClosedConnections in the user code. Automatic cleaning is performed once per
+ * second.
```

Comment:
Let's add 2 spaces in the beginning of each line to align with the bullet point?

Reply:
fixed.